onadata/apps/api/tools.py

```diff
@@ -172,6 +173,11 @@ def remove_user_from_organization(organization, user):
     owners_team = get_organization_owners_team(organization)
     remove_user_from_team(owners_team, user)
 
+    role = get_role_in_org(user, organization)
+    # Remove user from all org projects
+    for project in organization.user.project_org.all():
+        ShareProject(project, user.username, role, remove=True).remove_user()
```

Comment:
Should you be calling `ShareProject(project, user.username, role, remove=True).save()`, it will have the same effect and do any other cleanup required.